setup.cfg

```diff
@@ -63,3 +61,25 @@ build-dir = docs/build
 extend-ignore = SFS301 # Allow f-strings
 docstring-convention = numpy
 max-pos-args = 2
+
+[options.extras_require]
+glofas =
+    cdsapi
+    cfgrib
+full =
+    %(glofas)s
+test =
+    %(full)s
+    pytest
+    pytest-cov
+    pytest-mock
+    tox
+doc =
```

Comment:
Are there not more sphinx packages required with this new setup?

Reply:
`extras_require` already comes with the sphinx extension installed, so that's activated in `conf.py`